tests/factories/contacts/cht/clinic.js

```diff
@@ -0,0 +1,19 @@
+const Factory = require('rosie').Factory;
+
+const parent = { 
+  _id: 'hc1',
+  parent: {
+    _id: 'dist1'
+  } 
+};
+
+Factory.define('clinic')
+  .attr('_id','clinic1')
+  .attr('parent', parent)
+  .attr('type', 'clinic')
+  .attr('is_name_generated', 'true')
+  .attr('name', 'Househould 1')
+  .attr('external_id', '')
+  .attr('notes', '')
+  .attr('place_id', '19770')
+  .attr('reported_date', Date.now());
```

Comment:
The `new Date()` is going to run once, when this file is `require`d, and that date value will be used for all objects generated by this factory. What I think you actually want is this:
```suggestion
  .attr('reported_date', () => new Date());
```
...then that function will be executed each time and you'll get a new date each time.